.pre-commit-config.yaml

```diff
@@ -39,4 +39,9 @@ repos:
     exclude: '3rdparty/'
     args: ["--quiet"]
 
+- repo: https://github.com/jorisroovers/gitlint
```

Comment:
So what's this going to enforce? Just the title-match-regex? Can you add a short summary of the constraints to the dev guide? https://docs.zeek.org/projects/spicy/en/latest/development/coding-style.html#commit-messages

(And while you are at it: We can remove "Avoid rebases once pushed upstream", the current model works pretty well.)

Reply:
This will mainly catch overly long lines, and an incorrect title format (e.g., not starting with a capital letter, or not ending in `.`). I called out the tiltle part in the documentation.